### ISSUE TITLE
az network bastion tunnel exception

### ISSUE THREAD

**Issue by iseliweb** [Azure/azure-cli#24561]
Command throws a socket access error while trying to create a tunnel to a VM resources.

**Command Name**
`az network bastion tunnel`

**Errors:**
```
The command failed with an unexpected error. Here is the traceback:
[WinError 10013] An attempt was made to access a socket in a way forbidden by its access permissions
Traceback (most recent call last):
  File "D:\a\_work\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\knack/cli.py", line 233, in invoke
  File "D:\a\_work\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/core/commands/__init__.py", line 663, in execute
  File "D:\a\_work\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/core/commands/__init__.py", line 726, in _run_jobs_serially
  File "D:\a\_work\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/core/commands/__init__.py", line 697, in _run_job
  File "D:\a\_work\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/core/commands/__init__.py", line 333, in __call__
  File "D:\a\_work\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/core/commands/command_operation.py", line 121, in handler
  File "D:\a\_work\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/command_modules/network/custom.py", line 8411, in create_bastion_tunnel
  File "D:\a\_work\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/command_modules/network/custom.py", line 8397, in get_tunnel
  File "D:\a\_work\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/command_modules/network/tunnel.py", line 59, in __init__
PermissionError: [WinError 10013] An attempt was made to access a socket in a way forbidden by its access permissions

```

## To Reproduce:
- `az network bastion tunnel --name {} --resource-group {} --target-resource-id {} --resource-port {} --port {}`

Note that 'az network bastion rdp' works, error thrown only for tunneling.

## Expected Behavior
Tunnel creation

## Environment Summary
```
Windows-10-10.0.19044-SP0
Python 3.10.8
Installer: MSI

azure-cli 2.42.0
```
## Additional Context
-

**Comment by iseliweb** [comment]
net stop hns & net start hns finally resolved the issue. Seems that the port was still blocked due to a previous session.

**Comment by Msouza91** [comment]
Sorry for reopening this, but the fix proposed by OP didn't solve the issue for me, I've checked in resmon and the port I'm trying to use for the tunnel isn't currently in use by my device. I tried different ports as well to no avail.
It started happening after I upgraded to 2.45

I have tried uninstalling the bastion extension, the az cli, downgrading to 2.44.1, restarted my computer and network devices countless times, still I can't open a tunnel to my VM using this method.
